exporters/inmemory/build.gradle

```diff
@@ -11,6 +11,8 @@ ext.moduleName = "io.opentelemetry.exporters.inmemory"
 dependencies {
     api project(':opentelemetry-sdk')
 
+    implementation libraries.guava
+
```

Comment:
This felt like an ok dependency to introduce on the exporters given its usage elsewhere. While we don't use Guava in the public signatures of anything, we do use it to build completed futures internally.

Reply:
definitely for the in-memory exporter, this is 100% fine.

Reply:
It will no longer be required anywhere if I contribute to apply my CompletableExportStatus.